src/main/resources/db/migration/V3__.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE message
+    ADD last_changed date NULL;
+
+ALTER TABLE user
+DROP
+COLUMN github_id;
+
+ALTER TABLE user
+    ADD github_id INT NULL;
```

Comment:
This part of the migration is because me and Katti fucked around at first with making the githubId a Long in the entity-class. It was causing issues until we changed it to Integer, but we didn't make another migration(for some reason).

When testing this, you might need to delete your DB and just let the application c reate it again. I got errors because of this:p